phing/tasks/deploy.xml

```diff
@@ -84,6 +84,12 @@
     <phingcall target="target-hook:invoke">
       <property name="hook-name" value="post-deploy-build"/>
     </phingcall>
+    <if>
+      <isset property="simplesamlphp"/>
```

Comment:
@grasmash the only problem I see with that is if a project has implemented its SimpleSAML installation some other way. The simplesamlphp_auth module or the simplesamlphp library could be in composer.json.